src/styles/product-card.scss

```diff
@@ -0,0 +1,73 @@
+.product-card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
```

Comment:
Better to set this one  time for all elements under * selector in normalize or main styles file

Reply:
we had a lot of discussion up here about it, and i prefer this variant